src/libcommonserver/io/iohelper.cpp

```diff
@@ -155,6 +155,11 @@ bool IoHelper::_isExpectedError(IoError ioError) noexcept {
   \return true if no error occurred or if the error is expected, false otherwise.
 */
 bool IoHelper::_setTargetType(ItemType &itemType) noexcept {
+    if (itemType.targetPath.empty()) {
+        itemType.targetType = NodeTypeUnknown;
+        return true;
```

Comment:
Should we really return true? An item with type `NodeTypeUnknown`is not a normal behavior

Reply:
This can happen when the target path of a link is empty (dangling link), this is not an error.